src/qt_gui/discord_rpc_handler.cpp

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include <cstring>
+#include <ctime>
+#include "src/qt_gui/discord_rpc_handler.h"
+
+namespace DiscordRPCHandler {
+
+void RPC::init() {
+    DiscordEventHandlers handlers{};
+    std::string discordAppId = "1290207945476280360";
+    Discord_Initialize(discordAppId.c_str(), &handlers, 1, nullptr);
```

Comment:
App ID can simply be directly on the function argument, no need to initialize a std::string

Reply:
If that becomes the case later on, we can always change it back.